pwiz_tools/Skyline/TestData/TestData.csproj

```diff
@@ -224,10 +224,6 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <None Include="..\app.config">
-      <Link>app.config</Link>

```

Comment:
This interesting change was needed to get the code coverage build working on TeamCity: otherwise I got a userSettings configuration error in some of the TestFunctional tests (not TestData, which is the odd part). Removing it seems to have had no other impact as far as I can tell.